expression/chunk_executor.go

```diff
@@ -263,3 +260,28 @@ func VectorizedFilter(ctx sessionctx.Context, filters []Expression, iterator *ch
 	}
 	return selected, nil
 }
+
+// VectorizedFilterOneRow applies a list of filters to a row.
```

Comment:
better explain the meaning of return values.

Reply:
`VectorizedFilterOneRow` maybe not a good method name. 

Based on the method body, it does two things:

1. apply filters to these columns whose type is int 
2. if not, we further it `EvalBool` to these columns. 

`VectorizedFilterOneRow` suggests we apply all kind of filters to these columns.